cluster/gce/util.sh

```diff
@@ -58,6 +61,9 @@ function set-node-image() {
   elif [[ "${NODE_OS_DISTRIBUTION}" == "debian" ]]; then
     NODE_IMAGE=${KUBE_GCE_NODE_IMAGE:-${CVM_VERSION}}
     NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-google-containers}
+  elif [[ "${NODE_OS_DISTRIBUTION}" == "ubuntu" ]]; then
+    NODE_IMAGE=${KUBE_GCE_NODE_IMAGE}
```

Comment:
This shouldn't be necessary either, as it's redundant with https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-default.sh#L66